_posts/2021-04-07-upgrade-improvements.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Kafka upgrade improvements"
+date: 2021-04-07
+author: jakub_stejskal
+---
+The upgrade of Strimzi and Kafka is not a trivial process.
+In Strimzi 0.24 we introduced several improvements to make the process easier.
+This post shows you how the upgrade process has changed, and what you should do to keep your cluster working without issues.
+
+<!--more-->
+
+### Upgrade improvements
+
+Before release 0.24, the upgrade process required multiple steps.
+During upgrade, users had to go through all released versions between the current version and the target version.
+For example if you wanted to upgrade from 0.18 to 0.20, the recommended approach was to install 0.19 first, wait for a rolling update, and set new values for Kafka `version`, `log.message.format.version` and `inter.broker.protocol.version`. 
+And then repeat the same steps for 0.20.
+This made the upgrades hard to execute fully automatically. For example, when installing Strimzi from the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator).
+The rolling update of Kafka cluster was also time-consuming, as upgrading through several versions can take a long time.
+
+#### Process changes
+
+Now, the process is a bit more friendly.
+The main improvement is you don't need to upgrade across all Strimzi versions, For instance, you can upgrade from 0.24 directly to latest without upgrade through any other release like 0.25.
```

Comment:
_latest_ is a bit unclear, 0.25 does not exist yet. I think we need to write around it without using the versions.